src-overlay-ui/styles.scss

```
@@ -0,0 +1,81 @@
+:root {
+  --color-vrchat-status-blue: #409eff;
+  --color-vrchat-status-green: #67c23a;
+  --color-vrchat-status-orange: #fd9200;
+  --color-vrchat-status-red: #f56c6c;
+}
+
+* {
+  box-sizing: border-box;
+  letter-spacing: 0.025em;
+  font-family:
+    'Poppins', 'Noto Sans JP', 'Noto Sans KR', 'Noto Sans TC', 'Noto Sans SC', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)